providers/os/connection/snapshot/blockdevices.go

```diff
@@ -225,6 +234,18 @@ func (device BlockDevice) GetPartitions(includeBoot bool, includeMounted bool) (
 	return partitions, nil
 }
 
+func mapLVM2Partitions(part BlockDevice) (partitions []*PartitionInfo) {
+	for _, p := range part.Children {
+		devFsName := "/dev/mapper/" + p.Name
+		partitions = append(partitions, &PartitionInfo{
```

Comment:
nit: can we make a blockdevice -> partitioninfo func? i think we use this in more places than here